pkg/daemon/ceph/client/pool.go

```diff
@@ -387,10 +393,16 @@ func CreateReplicatedPoolForApp(context *clusterd.Context, clusterInfo *ClusterI
 }
 
 func createTwoStepCrushRule(context *clusterd.Context, clusterInfo *ClusterInfo, clusterSpec *cephv1.ClusterSpec, ruleName string, pool cephv1.PoolSpec) error {
```

Comment:
Shall we rename this?
```suggestion
func createStretchCrushRule(context *clusterd.Context, clusterInfo *ClusterInfo, clusterSpec *cephv1.ClusterSpec, ruleName string, pool cephv1.PoolSpec) error {
```